Utilities/ci-build

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eu
+
+__dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+cd "${__dir}/.."
+echo "Current directory is ${PWD}"
+
+CONFIGURATION=debug
+
+set -x
+
+swift build -c $CONFIGURATION
+# swift test -c $CONFIGURATION --parallel --enable-test-discovery
```

Comment:
No testing?

Reply:
I was hitting that crash in blake and I got lazy to type switch configuration to release :p This is just to configure/test the bots, we can change it once the bots are up.